kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerLauncher.kt

```diff
@@ -54,6 +72,30 @@ object JazzerLauncher {
         exitProcess(0)
     }
 
+    private fun Throwable.filter(): Throwable {
```

Comment:
Why is this method named `filter`? It seems like what it mainly does is just creates a new copy of the exception and removes Jazzer-specific stack trace lines

Reply:
Fixed